src/main/java/io/github/railroad/moddedVersionFetcher/fabric/FabricVersionHelper.java

```diff
@@ -1,120 +1,117 @@
 package io.github.railroad.moddedVersionFetcher.fabric;
 
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
-import javax.xml.xpath.XPathExpression;
-import javax.xml.xpath.XPathExpressionException;
-
-import org.w3c.dom.Document;
-import org.xml.sax.SAXException;
-
-import com.google.gson.Gson;
-import com.google.gson.reflect.TypeToken;
-
 
 /*
  * @author Saksham4106
- * 
+ *
  * Used to get the info for making a fabric mod all automatically
- * 
- *TODO: I still need to fix the fapi and builder thing 
+ *
+ *TODO: I still need to fix the fapi and builder thing
  */
 public class FabricVersionHelper {
```

Comment:
I am sure there is a better way to do this, similar to how I did the forge thing in my pr, but this can be fixed later.

Reply:
There is, but I didn't really work on it. I would appreciate it if you could come up with something.

Reply:
Yeah i can do that!